### ISSUE TITLE
New tweets not receiving using streame 

### ISSUE THREAD

**Issue by sapkal-manish** [ttezel/twit#282]
Hi folks, 
I want to get tweets from my friend in real time, so I can show it in my app. for that I have written following code.

```
var Twit = require('twit');

var T = new Twit({
  consumer_key:         '...',
  consumer_secret:      '...',
  access_token:         '...',
  access_token_secret:  '...',
  timeout_ms:           60*1000,  // optional HTTP request timeout to apply to all requests.
});

var userids = '100001,100002,100003';
T.stream('statuses/filter', 
    {
        follow: userids,
        stall_warnings: true
    },
    function(stream) {
        stream.on('tweet', function (tweet) {
            console.log(tweet);
        });
    }
);  

```

"userid" mentioned here is an example only. Now I am already logged in one of above account and tweeting, and my app is running too. still I am not getting any data in _stream.on('tweet')_ 

I think I have explained it well so anyone can understand my problem.

**Comment by ttezel** [comment]
Hi @sapkal-manish, please take care to read the README before posting issues here (StackOverflow is a better general help forum); I have noticed from your previously opened issue that you're not using the API in the way the documentation tells you to use it :) - `T.stream()` does not accept a callback function; it returns an EventEmitter. The return value is what you want to listen for events on. This is how you use the Streaming API:
```js
var stream = T.stream('statuses/filter', { .. your params here .. });
stream.on('tweet', function (tweet) {
  console.log(tweet)
});
```

**Comment by sapkal-manish** [comment]
That I had already tried, still not getting result as expected. Here is the my code 

```
var followers = ['74685201','801961424','240649814','134758540','2391969325','385909720'];
var stream = T.stream('statuses/filter', { follow: followers}); //74685201  //track: ['narendra']
stream.on('tweet', function (tweet) {
    console.log(tweet);
});
```
Here first 2 ids (e.g. '74685201','801961424') is mine, I have created it for my testing. while my app is up and running, I have made some tweets from this accounts, still not getting tweet on my app. You are right, @ttezel, but I have posted this question on stackoverfollow first, then I had posted here, because no one given answer there. I thought, may be some bug has been here in this library, thatway I have raised issue here. sorry for my bed English. Hope you understand.
